cloudinit/config/cc_resolv_conf.py

```diff
@@ -26,12 +26,18 @@
 }
 
 MODULE_DESCRIPTION = """\
+Unless manually editing :file:`/etc/resolv.conf` is the correct way to manage
+nameserver information on your operating system, you do not want to use
+this module. Many distros have moved away from manually editing ``resolv.conf``
+so please verify that this is the preferred nameserver management method for
+your distro :file:`/etc/resolv.conf` before using this module.
+
+Note that using :ref:`network_config` is preferred to this module, when
+possible.
+
 This module is intended to manage resolv.conf in environments where early
 configuration of resolv.conf is necessary for further bootstrapping and/or
 where configuration management such as puppet or chef own DNS configuration.
-As Debian/Ubuntu will, by default, utilize resolvconf, and similarly Red Hat
```

Comment:
Debian/Ubuntu aren't supported in this module, Red Hat no longer defaults to sysconfig (also not sure how that was ever relevant - on rhel 7 default method was always to edit /etc/resolv.conf ifaik)